src/control/server/README.md

```diff
@@ -1,5 +1,11 @@
 # DAOS Server (control-plane)
 
+## Workflow
+
+Control plane server (`daos_server`) instances will open and listen on gRPC channel for requests from control plane client applications. Administrators can perform provisioning operations on network and storage hardware through control plane [`dmg`](../dmg) management tool and when ready to start data plane, calling `dmg storage format` will enable the server to be formatted (superblock written) and start the data plane.
+
+![Server format diagram](/doc/graph/server_format_flow.png)
```

Comment:
Noticed a minor typo in the diagram: "Fotmat storage"

Reply:
annoying, I will redo